### DISCUSSION TITLE
Swipe the song right to add to queue

### DISCUSSION THREAD

**Discussion by FL1kkeR** [z-huang/music#1496] [Ideas]
Hello Zion, would it be possible to add a feature similar to Spotify where when you swipe a song to the right (from the playlist) it will be added to the queue

It's explained in this picture!

![NextPit-how-to-spotify-queue-1-w782.png](https://github.com/user-attachments/assets/7bca4d3e-a088-491a-91f8-c49f0e797e0f)

**Comment by z-huang** [comment]
Yes, I plan to add it in the near future.

**Reply by FL1kkeR** [reply]
Thanks boss
